tests/foreman/cli/test_errata.py

```diff
@@ -80,113 +74,107 @@
 from robottelo.constants.repos import FAKE_6_YUM_REPO
 from robottelo.constants.repos import FAKE_9_YUM_REPO
 from robottelo.decorators import skip_if_not_set
-from robottelo.test import CLITestCase
 from robottelo.vm import VirtualMachine
 
+
 ERRATUM_MAX_IDS_INFO = 10
 
+CUSTOM_REPO_URL = FAKE_9_YUM_REPO
+CUSTOM_PACKAGE = FAKE_1_CUSTOM_PACKAGE
+CUSTOM_ERRATA_ID = FAKE_2_ERRATA_ID
+CUSTOM_PACKAGE_ERRATA_APPLIED = FAKE_2_CUSTOM_PACKAGE
+
 
+@skip_if_not_set('clients', 'fake_manifest')
+@pytest.mark.skipif((not settings.repos_hosting_url), reason='Missing repos_hosting_url')
 @pytest.mark.run_in_one_thread
-class HostCollectionErrataInstallTestCase(CLITestCase):
+class TestHostCollectionErrataInstallTest:
```

Comment:
I think you can remove the `'Test'` at the end of this class name.

Reply:
right, and I have removed the classes now too